.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+permissions:
+    contents: write
+
+on:
+    push:
+        tags:
+            - "*"
+
```

Comment:
you can also set all env variables here so that you don't have to specify on each step

e.g.

```
env:
    NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
    GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```